docs/Basics/Quick Start - Forge.md

```diff
@@ -0,0 +1,55 @@
+# Quick Start: Dissonance with Forge Networking
+
+> This Quick Start guide is for those of you integrating Dissonance into a game with **Forge Networking**.
+
+This tutorial will guide you through the steps required to get a basic Dissonance setup working in your project. By the end of this tutorial, you will having working voice comms with all users talking in a global chat room.
+
+Before beginning this tutorial, please refer to the [installation guide](/Basics/Getting Started.md) to learn how to install Dissonance into your project.
+
+A demo scene for this tutorial can be found in the `Dissonance/Integrations/Forge/Demo` folder.
+
+## Step 1: Dissonance Comms Entity
+
+> Dissonance runs mostly from a single entity, which should be placed somewhere near the root of your scene. This entity contains the main `DissonanceComms` behavior, together with the Forge networking script. For this quick start guide, we will also attach transmission and receipt trigger scripts to this entity.
```

Comment:
> entity

Unity seems to prefer referring to entities as 'game objects'. I've made sure to refer to them as such in everything I've written. We want consistency so we either want to change your docs to use 'game object' or mine to use 'entity'.